Makefile

```diff
@@ -32,9 +39,15 @@ ta:
 			  $@
 
 .PHONY: clean
+ifneq ($(wildcard $(TA_DEV_KIT_DIR)/host_include/conf.mk),)
 clean:
 	$(q)$(MAKE) -C host/xtest O=$(out-dir)/xtest q=$(q) $@
 	$(q)$(MAKE) -C ta O=$(out-dir)/ta q=$(q) $@
+else
+clean:
+	$(q)echo "TA_DEV_KIT_DIR is not correctly defined - Remove $(out-dir)"
+	$(q)rm -rf $(out-dir)
```

Comment:
It is not possible to clean without the devkit.  So either "nothing" or Jerome's proposal is fine with me.